src/store/actions.js

```diff
@@ -368,6 +368,7 @@ export default {
 			originalSendAt = data.sendAt
 			const message = {
 				...data,
+				editorBody: data.body.value,
```

Comment:
drop this line to fix https://github.com/nextcloud/mail/pull/6372#issuecomment-1144910893

`data` has a `editorBody` property with the correct value, the HTML markup

In its current state the plain text is saved as editor body:
![Bildschirmfoto vom 2022-06-08 12-44-26](https://user-images.githubusercontent.com/1374172/172601278-0b82764a-1d4e-492e-8c99-cc5581fc6147.png)

Reply:
Thanks, fixed.